unit-tests/src/org/commcare/android/util/CaseLoadUtils.java

```diff
@@ -44,7 +42,7 @@ public static EntitySelectActivity launchEntitySelectActivity(String command) {
 
         // start the entity select activity
         return Robolectric.buildActivity(EntitySelectActivity.class)
-                .withIntent(entitySelectIntent).create().start().resume().get();
+                .withIntent(entitySelectIntent).setup().get();
```

Comment:
@amstone326 the culprit, as far as I can tell. Definitely took a bit to track down, but I guess it is important to call `visible()` on robolectric activities before trying to interact with them in certain ways: http://robolectric.org/activity-lifecycle/
(`setup()` calls `visible()`)